code/modules/power/reactor_stats.dm

```diff
@@ -112,7 +112,7 @@
 		if(G.carbon_dioxide) .["Carbon Dioxide|mols"] = G.carbon_dioxide
 		if(G.nitrogen) .["Nitrogen|mols"] = G.nitrogen
 
-		.["Pressure|Pa"] = MIXTURE_PRESSURE(G)
+		.["Pressure|Pa"] = MIXTURE_PRESSURE(G)*1000
```

Comment:
```suggestion
		.["Pressure|Pa"] = MIXTURE_PRESSURE(G) / (1 KILO)
```
would make more sense I think

Reply:
Seemed to produce 0, but I found other code using that similar macro and got it working